src/Web/WebApp/Components/Common/Cards/CreateEntityCard.razor.css

```
@@ -0,0 +1,25 @@
+}
+
+.hover-mud-border-primary:hover {
+    border-color: var(--mud-palette-primary) !important;
+}
+
+.hover-bg-surface-container:hover {
+     background-color: var(--mud-palette-surface) !important;
+}
+
+.mud-border-lines-default {
+    border-color: rgba(255, 255, 255, 0.12);
```

Stylelint (17.2.0)

[error] Expected "0.12" to be "12%" (alpha-value-notation)

(alpha-value-notation)